client/elements/sc-page-dictionary.html

```diff
@@ -217,25 +218,30 @@ <h3>Similar Spelling</h3>
                           'pts': 'PTS Pali English Dictionary'
                       }
                   },
-                  // The below two arrays are just temporary mockups.
-                  dictonaryAdjacent: {
-                      type: Array,
-                      value: ['dhamita','dhamitvā','dhameti','dhamenta','dhamesi',
-                      'dhamma','dhammakathā','dhammakathika','dhammakamma','dhammakaraka']
+                  adjacent: {
+                      type: Boolean,
+                      value: true
                   },
-                  dictionarySimilar: {
-                      type: Array,
-                      value: ['dhamma','dhammā','adhamma','dhamman','dhammī','ghamma',
-                      'dhama','damma','uddhamma','opamma']
-                  }
+                  glossaryList: Object,
+                  dictionaryAdjacent: Array,
+                  dictionarySimilar: Array
               }
           }
 
+          static get observers() {
+              return [
+                  '_getGlossaryItems(glossaryList,adjacentReturns,adjacent)',
+                  '_getGlossaryItems(glossaryList,similarReturns)'
```

Comment:
When `glossaryList` changes, `_getGlossaryItems` will get called twice.

Reply:
I tried that but then it is called before the glossaryList is actually loaded properly and not again after it is loaded.

Reply:
Oh, I see. Yeah, that would be a problem. I think you can fix it by calling `_getGlossaryItems()` once after it's ben loaded, inside`_glossaryLoaded()`. The observers would then react on subsequent changes of `adjacentReturns`, `adjacent` and `similarReturns`.

If it still doesn't work, we can leave it like that, as long as calling `_getGlossaryItems` twice doesn't incur a significant performance hit.

Reply:
I tried that too but then it gets called before the inputArrays (`adjacentReturns` & `adjacentSimilar`) are loaded. 
It should not be a significant performance hit because `adjacentReturns` & `adjacentSimilar` are very small. `glossaryList` is pretty big but only gets loaded once and never changes after that.